framework/src/transfers/MultiAppTransfer.C

```diff
@@ -172,6 +173,66 @@ MultiAppTransfer::checkMultiAppExecuteOn()
                        "execute_on flags"));
 }
 
+void
+MultiAppTransfer::checkPostProcessorExecuteOn(std::string to_check)
+{
+  std::vector<std::shared_ptr<MultiApp>> multi_apps;
+  if (_from_multi_app)
+    multi_apps.push_back(_from_multi_app);
+  if (_to_multi_app)
+    multi_apps.push_back(_to_multi_app);
+  for (std::shared_ptr<MultiApp> multi_app : multi_apps)
+  {
+    // check subapps
+    for (unsigned int i = 0; i < multi_app->numGlobalApps(); i++)
+    {
+      // If we do not have this app, we skip
+      if (!multi_app->hasLocalApp(i))
+        continue;
+      FEProblemBase & sub_problem = multi_app->appProblemBase(i);
+      checkAgainstPPExec(to_check, sub_problem);
+    }
+
+    // check main app
+    FEProblemBase & main_problem = multi_app->problemBase();
+    checkAgainstPPExec(to_check, main_problem);
+  }
+}
+
+void
+MultiAppTransfer::checkAgainstPPExec(std::string to_check, FEProblemBase & problem)
+{
+  // It's possible that the Transfer is going to/from a pp/vpp in a subapp. Therefore, if it doesn't
+  // have the pp/vpp name in this apps problem, return, as this behavior is checked in the subapp
+  // portion of checkPostProcessorExecuteOn.
+  if (!problem.hasUserObject(to_check))
+    return;
+
+  const ExecFlagEnum & object_execute_on = problem.getUserObjectBase(to_check).getExecuteOnEnum();
+  const ExecFlagEnum & transfer_execute_on = _pars.get<ExecFlagEnum>("execute_on");
```

Comment:
use the interface routine, avoid calling parameter stuff when possible it s wordy

Reply:
the SetupInterface, has getExecuteOnEnum.

Is this not an interface for MultiAppTransfer?

Reply:
Oh, so just ```this->getExecuteOnEnum()?``` I mean, it's probably possible, but seems a bit silly for an entire commit, that's only like 10 characters shorter

Reply:
well you ll squash commits afterwards.
It s not just length. If we consistently use interfaces, when we improve them we improve things everywhere